src/components/Card.js

```diff
@@ -1,9 +1,13 @@
+import Link from 'next/link';
 function Card({exercise}) {
 	return (
 		<article>
 			<h2>{exercise.name}</h2>
 			<h3>{exercise.sanscritName}</h3>
 			<p>{exercise.about}</p>
+			<Link href={{pathname: '/wirkung', query: {keyword: exercise.id}}}>
+				<a>wirkung</a>
+			</Link>
```

Comment:
Diese Änderung bezieht sich auf meinen Kommentar in der `wirkung.js`👀

```suggestion
			<Link href={`/wirkung/${exercise.id}`}>
				<a>wirkung</a>
			</Link>
```